mscore/pagesettings.h

```diff
@@ -42,6 +42,7 @@ class PageSettings : public AbstractDialog, private Ui::PageSettingsBase {
       void blockSignals(bool);
       void applyToScore(Score*);
       void setMarginsMax(double);
+      QPushButton* buttonApplyToAllParts = new QPushButton(tr("Apply to all Parts"));
```

Comment:
Is this needed? instanciating an object in a header file?

Reply:
Moved to the Constructor as requested. That's clearly better. Thanks.